src/homeworks/test_47/index.ts

```diff
@@ -0,0 +1,11 @@
+
+type dataInput = Record<"year" | "month" | "day", number>
```

Comment:
I tipi dovrebbero cominciare con la lettera maiuscola.
```suggestion
type DataInput = Record<"year" | "month" | "day", number>
```